installer/windows/Product.wxs

```diff
@@ -83,19 +83,26 @@
     <Property Id="STORJ_STORAGEDIR">STORAGEDIR</Property>
     <Property Id="STORJ_STORAGE">1.0</Property>
     <Property Id="STORJ_BANDWIDTH">2.0</Property>
+    <Property Id="WIXUI_EXITDIALOGOPTIONALCHECKBOXTEXT" Value="Launch SNOBoard" />
```

Comment:
Let's change the text to "Open dashboard". We call it "SNOboard" internally, but users who just install the product most probably won't be familiar with this term.